### ISSUE TITLE
[Bug]: RPC throws NoTransformationFoundException for functions that returns void

### ISSUE THREAD

**Issue by ferhatwi** [supabase-community/supabase-kt#106]
### General Info

- [X] I checked for similar bug report

### Version(s)

0.9.0

### Target(s)

Android

### What happened? (include your code)

Calling a supabase function that returns void, throws NoTransformationFoundException. 
It probably tries to convert result body to `JsonElement`. Since result has no content, `JsonElement` doesn't get created.

Setting `head` to `true` or `false` doesn't change the output, still throws.

### Relevant log output (optional)

```shell
Exception in thread "main" io.ktor.client.call.NoTransformationFoundException: No transformation found: class io.ktor.utils.io.ByteBufferChannel -> class kotlinx.serialization.json.JsonElement
with response from https://PROJECT-URL.supabase.co/rest/v1/rpc/void_test:
status: 204 No Content
```

**Comment by jan-tennert** [comment]
Should be fixed in 0.9.1

**Comment by ferhatwi** [comment]
Thanks!
